client/packages/common/src/intl/locales/en/coldchain.json

```
@@ -11,7 +11,8 @@
   "button.add-new-program": "New Program",
   "label.add-new-vaccine-course": "Add vaccine course",
   "heading.schedule": "Schedule",
-  "heading.vaccine-details": "Vaccine course details",
+  "heading.create-vaccine-course": "Create new vaccine course",
```

:nit slightly tautologous; if I'm creating then it is probably 'new'
```suggestion
  "heading.create-vaccine-course": "Create vaccine course",
```